docs/config.ini

```diff
@@ -18,29 +18,29 @@ default = index.html
 home = getting_started/get_started.html
 outline = developing_course/course_outline.html
 unit = developing_course/course_units.html
-updates = building_course/handouts_updates.html
-pages = building_course/pages.html
-files = building_course/course_files.html
-textbooks = building_course/textbooks.html
-schedule = building_course/setting_up_student_view.html
-grading = building_course/establish_grading_policy.html
-team_course = building_course/creating_new_course.html#add-course-team-members
+updates = course_assets/handouts_updates.html
+pages = course_assets/pages.html
+files = course_assets/course_files.html
+textbooks = course_assets/textbooks.html
+schedule = set_up_course/setting_up_student_view.html
+grading = grading/index.html
+team_course = set_up_course/course_staffing.html#add-course-team-members
 team_library = creating_content/libraries.html#give-other-users-access-to-your-library
 advanced = index.html
-checklist = building_course/creating_new_course.html#use-the-course-checklist
+checklist = set_up_course/creating_new_course.html#use-the-course-checklist
 import_library = creating_content/libraries.html#import-a-library
-import_course = building_course/export_import_course.html#import-a-course
+import_course = releasing_course/export_import_course.html#import-a-course
 export_library = creating_content/libraries.html#export-a-library
-export_course = building_course/export_import_course.html#export-a-course
+export_course = releasing_course/export_import_course.html#export-a-course
```

Comment:
did this change to the location of export & import make it into the open_edx_course_authors guide?